components/SingleEntry/Entry.js

```diff
@@ -15,15 +15,21 @@ import { FaRegEye } from "react-icons/fa6";
 import { useState } from "react";
 import { AnimatePresence, motion } from "framer-motion";
 import { useSession } from "next-auth/react";
+import { useSphereState } from "../ContextProviders/SphereStateProvider/SphereStateProvider";
+import { useEffect } from "react";
 
 export default function Entry({ id }) {
+  const { sphereState, handleSphereState } = useSphereState();
```

Comment:
```suggestion
  const { handleSphereState } = useSphereState();
```
same here but reverse, are you using the sphereState in this file?

Reply:
so true!